cmd/list.go

```diff
@@ -150,4 +156,5 @@ func init() {
 	RootCmd.AddCommand(listCmd)
 
 	listCmd.Flags().BoolVar(&listOpts.base64encode, "base64", false, "Show values as base64-encoded string")
+	listCmd.Flags().BoolVar(&listOpts.allnamespaces, "all-namespaces", false, "Show values in All Namespaces")
```

Comment:
How about adding short `-A` like the one of `kubectl get`？

>   -A, --all-namespaces=false: If present, list the requested object(s) across all namespaces. Namespace in current
context is ignored even if specified with --namespace.

Reply:
It's nice idea 👍

Reply:
✅ https://github.com/wantedly/k8sec/pull/7/commits/247313899dfe60b2274fe628cf25478df9dcffa6